source/mslite-metadata.json

```diff
@@ -0,0 +1,2017 @@
+    "schema": {
+      "inputs": [
+        { "name": "input" }
+      ],
+      "outputs": [
+        { "name": "output" }
+      ],
+      "attributes": [
+        { "name": "axes", "type": "[int]"},
+        { "name": "keepDims", "type": "int", "default": 0 },
+        { "name": "mode", "type": "ReduceMode"},
+        { "name": "reduceToEnd", "type": "bool", "default": false },
+        { "name": "coeff", "type": "float", "default": 1.0 }
+      ]
+    }
+  },
+  {
+    "name": "Reshape",
+    "schema": {
+      "inputs": [
+        { "name": "input" },
+        { "name": "shape" }
+      ],
+      "outputs": [
+        { "name": "output" }
+      ],
+      "attributes": [
+        { "name": "format", "type": "Format" },
+        { "name": "shape", "type": "[int]" }
+      ]
+    }
+  },
+  {
+    "name": "Resize",
+    "schema": {
+      "inputs": [
+        { "name": "input" },
+        { "name": "size" }
+      ],
+      "outputs": [
+        { "name": "output" }
+      ],
+      "attributes": [
+        { "name": "format", "type": "Format" },
+        { "name": "method", "type": "ResizeMethod", "default": 0 },
+        { "name": "newHeight", "type": "long", "default": 0 },
+        { "name": "newWidth", "type": "long", "default": 0 },
+        { "name": "alignCorners", "type": "bool", "default": false },
+        { "name": "preserveAspectRatio", "type": "bool", "default": false }
+      ]
+    }
+  },
+  {
+    "name": "Return"
+  },
+  {
+    "name": "Reverse",
+    "schema": {
+      "inputs": [
+        { "name": "input" }
+      ],
+      "outputs": [
+        { "name": "output" }
+      ],
+      "attributes": [
+        { "name": "axis", "type": "[int]" }
+      ]
+    }
+  },
+  {
+    "name": "ReverseSequence",
+    "schema": {
+      "inputs": [
+        { "name": "input" },
+        { "name": "seq_lengths" }
+      ],
+      "outputs": [
+        { "name": "output"}
+      ],
+      "attributes": [
+        { "name": "seqAxis", "type": "int", "default": 0 },
+        { "name": "batchAxis", "type": "int", "default": 0 }
+      ]
+    }
+  },
+  {
+    "name": "Rfft",
+    "schema": {
+      "attributes": [
+        { "name": "fftLength", "type": "int", "default": 0 }
+      ]
+    }
+  },
+  {
+    "name": "ROIPooling",
+    "schema": {
+      "inputs": [
+        { "name": "input" }
+      ],
+      "outputs": [
+        { "name": "output" }
+      ],
+      "attributes": [
+        { "name": "pooledH", "type": "int", "default": 0 },
+        { "name": "pooledW", "type": "int", "default": 0 },
+        { "name": "scale", "type": "float", "default": 0.0 }
+      ]
+    }
+  },
+  {
+    "name": "Round",
+    "schema": {
+      "inputs": [
+        { "name": "x" }
+      ],
+      "outputs": [
+        { "name": "y" }
+      ]
+    }
+  },
+  {
+    "name": "Rsqrt",
+    "schema": {
+      "inputs": [
+        { "name": "x" }
+      ],
+      "outputs": [
+        { "name": "y" }
+      ]
+    }
+  },
+  {
+    "name": "Scale",
```

Comment:
`Scale` is missing `"category": "Layer"`, `inputs` and `outputs`.

<img width="143" alt="scale" src="https://user-images.githubusercontent.com/438516/96325882-c361f580-0fe0-11eb-9791-0bb097f739c3.png">